test/python/qobj/test_pulse_converter.py

```diff
@@ -304,7 +304,7 @@ def test_set_frequency(self):
         """Test converted qobj from FrameChangeInstruction."""
         instruction = SetFrequency(8.0, DriveChannel(0))
 
-        qobj = PulseQobjInstruction(name='setf', ch='d0', t0=0, frequency=8.0)
+        qobj = PulseQobjInstruction(name='setf', ch='d0', t0=0, frequency=8e-9)
```

Comment:
Would we fix the tests to that this is `qobj = PulseQobjInstruction(name='setf', ch='d0', t0=0, frequency=8.0)` and the pulse instruction is 8e9 please? This is more natural.

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4399/commits/c70dd7c3c4626d46a45195b5e47ccbc2348ce479.